<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Juego</class>
 <widget class="QMainWindow" name="Juego">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Othello</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos.qrc">
    <normaloff>:/iconos/othello13.png</normaloff>:/iconos/othello13.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-image: url(:/iconos/Fondo.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>521</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="botonJuegoNuevo">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	color: white;
	background-color: transparent;
    border-style: solid;
    border-width: 2px;
    border-radius: 12px;
    border-color: beige;
    font: bold 12px;
    min-width: 4em;
    padding: 3px;
} 
QPushButton:hover:!pressed
{
	border: 2px solid black;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>Nuevo juego</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="botonInstrucciones">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	color: white;
	background-color: transparent;
    border-style: solid;
    border-width: 2px;
    border-radius: 12px;
    border-color: beige;
    font: bold 12px;
    min-width: 4em;
    padding: 3px;
} 
QPushButton:hover:!pressed
{
	border: 2px solid black;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>Instrucciones</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewTablero">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>521</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTurno">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>24</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: beige;
	font: 75 24pt &quot;MS Shell Dlg 2&quot;;
	border-style: solid;
    border-width: 4px;
    border-radius: 12px;
    border-color: beige;
}</string>
    </property>
    <property name="text">
     <string>Turno negro</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResultado">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>70</y>
      <width>121</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: beige;
	background-color: green;
    border-style: solid;
    border-width: 6px;
    border-radius: 12px;
    border-color: beige;
    min-width: 4em;
	font: bold 18px;
    padding: 3px;
} </string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;Jugador negro:&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;Jugador blanco:&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPuntajeNegro">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>170</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	font: bold 14px;
	color: beige;
}</string>
    </property>
    <property name="text">
     <string>x 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPuntajeBlanco">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	font: bold 14px;
	color: beige;
}</string>
    </property>
    <property name="text">
     <string>x 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFichaNegra">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>169</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	image: url(:/iconos/fichaNegra.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelFichaBlanca">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>280</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	image: url(:/iconos/fichaBlanca.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>graphicsViewTablero</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>labelTurno</zorder>
   <zorder>labelResultado</zorder>
   <zorder>labelPuntajeNegro</zorder>
   <zorder>labelPuntajeBlanco</zorder>
   <zorder>labelFichaNegra</zorder>
   <zorder>labelFichaBlanca</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>
